sdk/Lusid.Sdk/Utilities/LusidApiFactoryBuilder.cs

```diff
@@ -19,16 +19,27 @@ public static ILusidApiFactory Build(string apiSecretsFilename)
         /// <summary>
         /// Create an ILusidApiFactory using the specified Url and Token Provider
         /// </summary>
-        public static ILusidApiFactory Build(string url, ITokenProvider tokenProvider, Dictionary<string, string> defaultHeaders = null)
+        public static ILusidApiFactory Build(string url, ITokenProvider tokenProvider, IDictionary<string, string> defaultHeaders = null)
         {
             // TokenProviderConfiguration.ApiClient is the client used by LusidApiFactory and is 
             // NOT thread-safe, so there needs to be a separate instance for each instance of LusidApiFactory.
             // Do NOT cache the LusidApiFactory instances (DEV-6922)
-            var config = new TokenProviderConfiguration(tokenProvider)
+            TokenProviderConfiguration config;
```

Comment:
i think its neater to write as below (avoids duplication)

```
  var config = new TokenProviderConfiguration(tokenProvider)
  {
      BasePath = url
  };
  if (defaultHeaders != null) {
    config.DefaultHeaders = defaultHeaders;
  }
```

Reply:
I knew there'd be a better way...